honeybadger-java/src/main/java/io/honeybadger/reporter/config/MapConfigContext.java

```diff
@@ -75,7 +75,7 @@
     /** System property key indicating the maximum number of attempts to submit an error.
      * see { @link HoneyBadgerReporter }
      */
-    public static final String HONEYBADGER_MAXIMUM_ERROR_REPORTING_ATTEMPTS_KEY =
+    public static final String HONEYBADGER_MAXIMUM_ERROR_REPORTING_RETRIES_KEY =
             "honeybadger.maximum_error_reporting_attempts";
```

Comment:
This needs to be changed too, right?

Reply:
Is there a way to have test coverage on this config value? The build is currently green, but it probably shouldn't be.